frontend/src/streamlist/DeleteStream.jsx

```diff
@@ -22,6 +25,8 @@ export default function DeleteStream(props) {
   };
   const confirmDelete = () => {
     StreamApi.deleteStream(deleteId);
+    history.push("/Streaming");
+    history.go(0);
```

Comment:
Glad `.go(0)` was added after some discussion otherwise the page wouldn't refresh!